content/YubiHSM2/Usage_Guides/YubiHSM_2_Windows_Deployment_Guide/.conf.json

```diff
@@ -0,0 +1,9 @@
+{
+  "order": [
+    "Introduction",
+    "Prerequisites_and_Preparations",
+    "Install_the_YubiHSM_Tools_and_Software"
```

Comment:
there's a comma missing at the end of line here, leading to the build failing

Reply:
Thanks for letting me know exactly what was wrong